cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControlConfiguration.java

```diff
@@ -79,6 +79,11 @@
     public static final String CRUISE_CONTROL_HARD_GOALS_CONFIG_KEY = "hard.goals";
     public static final String CRUISE_CONTROL_SELF_HEALING_CONFIG_KEY = "self.healing.goals";
     public static final String CRUISE_CONTROL_ANOMALY_DETECTION_CONFIG_KEY = "anomaly.detection.goals";
+    public static final String CRUISE_CONTROL_PARTITION_METRICS_WINDOW_MS_CONFIG_KEY = "partition.metrics.window.ms";
+    public static final String CRUISE_CONTROL_PARTITION_METRICS_WINDOW_NUM_CONFIG_KEY = "num.partition.metrics.windows";
+    public static final String CRUISE_CONTROL_BROKER_METRICS_WINDOW_MS_CONFIG_KEY = "broker.metrics.window.ms";
+    public static final String CRUISE_CONTROL_BROKER_METRICS_WINDOW_NUM_CONFIG_KEY = "num.broker.metrics.windows";
+    public static final String CRUISE_CONTROL_COMPLETED_USER_TASK_RETENTION_MS_CONFIG_KEY = "completed.user.task.retention.time.ms";
```

Comment:
As we have a `CruiseControlServerParameters` enum and these values are server configs I think I would make sense to put move these to that enum so there is a single source of truth for these Keys. Otherwise we now have to manage these in two places?

Reply:
I am very happy to move them there. I will dig out their default values as well.
Also, just those 4 vars, or more?

Reply:
Well ideally it would be all of them, you would have to move all the goals lists definitions as well and be careful to refactor all the tests that use those lists. 

@ppatierno wdyt about making the `CruiseControlServerParameters` enum the single source of truth for the CC server config keys *and their defaults*? Or should we just use the enum for the key names and define the defaults in the `CruiseControlConfiguration` class?

Reply:
I am fine to move everything there.